pubspec.yaml

```diff
@@ -177,63 +187,63 @@ flutter:
   uses-material-design: true
 
   assets:
-  - assets/icons/
-  - assets/illustrations/
-  - assets/images/
-  - packages/time_machine/data/cultures/cultures.bin
-  - packages/time_machine/data/tzdb/tzdb.bin
+    - assets/icons/
```

Comment:
There is something wrong with your formatter. The formatting was definitely correct here and below, please revert.

Reply:
This wasn't fixed.